### PR TITLE
Add in extra automation in advanced section

### PR DESCRIPTION
**Pull request by lukewest**
When HA starts up, it is vital that the correct tariff is set dependant upon the time. 
e.g. startup at 10h10 = peak
or startup at 22h10 = offpeak
If this is not correct then the "next_tariff" will be wrong.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/utility_meter.markdown [hunk 1]
```diff
@@ -154,8 +161,41 @@ automation:
...
+    - service: utility_meter.select_tariff
+      data_template:
+        entity_id:
+          - utility_meter.daily_energy
+          - utility_meter.monthly_energy
+        # tariff
+        # 09:00 - 21:00 = peak
+        # 21:00 - 09:00 = offpeak
+        tariff: >-
+          {% if (now().hour < 9) %}
```
Comment (lukewest) [inline source/_integrations/utility_meter.markdown:189]: strictly speaking the brackets are not needed

**Comment by frenck** [comment]
Duplicate of #16031?

Could you please clarify the two PRs?

**Comment by lukewest** [comment]
Its a duplicate. My browser crashed just at time of submission

The PR’s are to improve documentation to permit the use of a tariff switch.

I find it easier to set the correct tariff each time HA starts then use a timing automation to flip between. In my case I have a lot more switches in a day for my park/offpeak.
00:00 - 00:30 = offpeak
00:30 - 02:30 = peak
02:30 - 07:30 = offpeak
07:30 - 22:30 = peak
22:30 - 00:00 = offpeak

L

On 28 Dec 2020, at 13:43, Franck Nijhof <notifications@github.com> wrote:


Duplicate of #16031 <https://github.com/home-assistant/home-assistant.io/pull/16031>?

Could you please clarify the two PRs?

**Comment by frenck** [comment]
Please close the one that isn't relevant.

**Comment by lukewest** [comment]
Duplicate

**Comment by lukewest** [comment]
I’ve closed this one

L

On 28 Dec 2020, at 15:22, Franck Nijhof <notifications@github.com> wrote:


Please close the one that isn't relevant.

**Pull request closed without merging**